test/components/structures/TimelinePanel-test.tsx

```diff
@@ -817,7 +817,7 @@ describe("TimelinePanel", () => {
             client = MatrixClientPeg.safeGet();
 
             Thread.hasServerSideSupport = FeatureSupport.Stable;
-            room = new Room("roomId", client, "userId");
+            room = new Room("roomId", client, "userId", { pendingEventOrdering: PendingEventOrdering.Detached });
```

Comment:
Why is it needed?

Reply:
The test started failing - this is what we always use in react-sdk but it's still not the default in react-sdk, so we probably got away with it before because we were never checking for unsent messages in the test.